fence/resources/ga4gh/passports.py

```diff
@@ -118,40 +167,106 @@ def get_unvalidated_visas_from_valid_passport(passport, pkey_cache=None):
     return decoded_passport.get("ga4gh_passport_v1", [])
 
 
-def is_raw_visa_valid(raw_visa):
-    # check signature
-    # is a type we recognize?
-    return False
-
-
-def get_sub_iss_from_visa(raw_visa):
-    if not is_valid_visa(raw_visa):
-        raise Exception()
-
-    subject_id = None
-    issuer = None
-
-    # TODO
-
-    return subject_id, issuer
-
-
-def sync_valid_visa_authorization(visa):
-    # DOES NOT VALIDATE VISA
-
-    # syncs authz to backend
-
-    pass
-
+def validate_visa(raw_visa):
+    # TODO check that there is no JKU field in header?
+    decoded_visa = validate_jwt(
+        raw_visa,
+        attempt_refresh=True,
+        scope={"openid", "ga4gh_passport_v1"},
+        require_purpose=False,
+        issuers=config.get("GA4GH_VISA_ISSUER_ALLOWLIST", []),
```

Comment:
if you're using config you don't have to .get(), you can access directly and it will default to whatever is in the default config file. so just 
```suggestion
        issuers=config["GA4GH_VISA_ISSUER_ALLOWLIST"],
```